packages/fx-core/resource/package.nls.json

```diff
@@ -137,6 +137,22 @@
   "plugins.appstudio.manuallySubmitDescription": "Build the Teams app package. You will need to send it to your Teams administrator manually",
   "plugins.appstudio.autoPublishTip": "Install for your organization",
   "plugins.appstudio.autoPublishDescription": "Send your app to your Teams administrator for approval via the Teams App Portal.",
+  "plugins.bot.questionHostTypeTrigger.title": "Choose triggers",
+  "plugins.bot.questionHostTypeTrigger.placeholder": "Choose triggers",
+  "plugins.bot.questionHostTypeTrigger.error.emptySelection": "You have to select at least one trigger.",
+  "plugins.bot.questionHostTypeTrigger.error.hostTypeConflict": "Azure Functions and Azure App Service host types are in conflict.",
+  "plugins.bot.triggers.functionsTimer.label": "Timer Trigger",
+  "plugins.bot.triggers.functionsTimer.cliName": "timer",
```

Comment:
Nit: `cliName` is interesting. Why name it like this, any explanations?

Reply:
All the keys (label, cliName, etc) are directly from `OptionItem` of question model: 
https://github.com/OfficeDev/TeamsFx/blob/829454cccdaba14de331c13830d69c815c1a2454/packages/api/src/types.ts#L23-L48

It is for CLI --argument name